core/common/modules/connect/tests/connect-test.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Elementor\Core\Common\Modules\Connect\Tests;
+
+use Elementor\Core\Utils\Testing\Test;
+use Elementor\Plugin;
+
+class Connect_Test extends Test {
+	public function test_connect_response_is_200() {
+		// Arrange
```

Comment:
Add a filter for `wp_die_jsonp_handler` also use `register_shutdown_function`

Reply:
Done for `wp_die_jsonp_handler`. `register_shutdown_function` isn't needed in this case.